lib/core/res/errors.dart

```diff
@@ -0,0 +1,2 @@
+// Common Errors
+class NetworkException extends Error {}
```

Comment:
```suggestion
class NetworkException implements Exception {}
```
This is what you are supposed to do.

Reply:
done!